pocs/observatory.py

```diff
@@ -130,17 +131,23 @@ def primary_camera(self, cam):
 
     @property
     def current_observation(self):
+        if not self.can_observe:
+            raise error.NoObservation("No valid observations found.")
         return self.scheduler.current_observation
 
     @current_observation.setter
     def current_observation(self, new_observation):
+        if not self.can_observe:
+            raise error.NoObservation("No valid observations found.")
         self.scheduler.current_observation = new_observation
 
     @property
     def has_dome(self):
         return self.dome is not None
 
-
+    @property
+    def can_observe(self):
+        return self.scheduler is not None
```

Comment:
This is going to expand to include things like the camera, mount, etc. Let's set up the framework to test each of those and give appropriate message if not found.

This will end up something like:
```python
def can_observe(self):
   can_observe = True

    if can_observe and self.scheduler is None:
        self.logger.info(f'Scheduler not present, cannot observe')
        can_observe = False

    # Same thing for mount
    # Same thing for cameras
    # Etc.

    return can_observe
```

Also, docstrings. ;)